packages/ag-charts-community/src/chart/series/seriesAreaManager.ts

```diff
@@ -169,7 +170,11 @@ export class SeriesAreaManager extends BaseManager {
     }
 
     public setTabIndex(tabIndex: number) {
-        this.swapChain.setTabIndex(tabIndex);
+        // FIXME: Remove keyboard.tabIndex option in next major release
+        if (tabIndex !== 0 && tabIndex !== -1) {
+            Logger.warnOnce(`Tab indices other than 0 and -1 are deprecated`);
+        }
```

Comment:
No need for this, we're in a major release cycle so let's just put the new constraint in place and add `Breaking Changes` notes to the JIRA?

Reply:
I agree, but we need consensus with David before we can go ahead with that.

Reply:
Discussed offline.

Reply:
Removed the warning. The `tabIndex > 0` issue will be addressed in a separate PR.